deploy.sh

```diff
@@ -15,6 +15,8 @@ if [ -z ${D4M_HOME+x} ]; then
   echo "Not installing Graphulo JAR in D4M installation because D4M_HOME is not set";
 else
   cp $(echo target/graphulo-*.jar | tr ' ' '\n' | grep -v alldeps) "$D4M_HOME/lib"
+  cp -f target/graphulo-*-alldeps.jar $D4M_HOME/lib
```

Comment:
You probably don't want the `alldeps` jar in D4M because it is very big and contains all the external dependencies all in one jar.  We use the zip file to unzip the external dependencies separately. Did something break?